core/assets/jsons/save.json

```diff
@@ -1,50 +1,57 @@
-[
-  {
-    "name": "easyLevel",
-    "unlocked": true,
-    "completed": false,
-    "path": "levels/easyLevel.json"
-  },
-  {
-    "name": "mediumLevel",
-    "unlocked": false,
-    "completed": false,
-    "path": "levels/mediumLevel.json"
-  },
-  {
-    "name": "hardLevel",
-    "unlocked": false,
-    "completed": false,
-    "path": "levels/hardLevel.json"
-  },
-  {
-    "name": "level4",
-    "unlocked": false,
-    "completed": false,
-    "path": "levels/enemySurroundingPlayer.json"
-  },
-  {
-    "name": "level5",
-    "unlocked": false,
-    "completed": false,
-    "path": "levels/shooterAndPatrol.json"
-  },
-  {
-    "name": "level6",
-    "unlocked": false,
-    "completed": false,
-    "path": "levels/onlypatrol.json"
-  },
-  {
-    "name": "easyTest",
-    "unlocked": false,
-    "completed": false,
-    "path": "levels/easy.json"
-  },
-  {
-    "name": "mediumTest",
-    "unlocked": false,
-    "completed": false,
-    "path": "levels/medium.json"
-  }
-]
\ No newline at end of file
+{
+  "levels": [
+    {
+      "name":       "easyLevel",
+      "unlocked":   true,
+      "completed":  true,
+      "path":       "levels/easyLevel.json"
+    },
+    {
+      "name":       "mediumLevel",
+      "unlocked":   true,
```

Comment:
This file should NOT be set to all unlocked. This determines what is initially unlocked in the JAR

Reply:
You should add your level, but leave everything locked except the first tutorial level